### PR TITLE
Add conda recipe for centralized-pre-commit-conf.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Comment by couger01** [comment]
Please consider removing centralized-precommmit-conf source code from the repo and changing the meta.yaml to include a `git_source` line that downloads the source code instead.

**Review by couger01 — CHANGES_REQUESTED** [review]
Please consider removing pre-commit-centralized-conf source code and changing meta.yaml to include  line to download the source code when building the recipe

**Comment by wvreeven** [comment]
Thanks for the very useful comment. Done.

File: centralized-pre-commit-conf/conda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - pre-commit-conf = centralized_pre_commit_conf.main:run
+
+test:
+  requires:
+    - pytest-cov
+    - pytest-vcr
+    - python-coveralls
+    - coverage
+    - coveralls
+    - pylint =2.15.4
```
Comment (couger01) [inline centralized-pre-commit-conf/conda/meta.yaml:24]: Add 

source_files:
  - tests

File: centralized-pre-commit-conf/conda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  imports:
+    - centralized_pre_commit_conf
+
```
Comment (couger01) [inline centralized-pre-commit-conf/conda/meta.yaml:27]: Add 

commands:
  - pre-commit-conf -h
  - tests

**Review by couger01 — CHANGES_REQUESTED** [review]

File: centralized-pre-commit-conf/conda/meta.yaml (hunk 1 above)
Comment (wvreeven) [reply centralized-pre-commit-conf/conda/meta.yaml:24]: Done. By the way, you can add source code here too by using triple backtciks button. So I added
```
  source_files:
    - tests
```

File: centralized-pre-commit-conf/conda/meta.yaml (hunk 2 above)
Comment (wvreeven) [reply centralized-pre-commit-conf/conda/meta.yaml:29]: I did a slightly different change that works too and is more in line with our usual conda recipes:
```
  commands:
    - pytest
```

**Review by couger01 — APPROVED** [review]
LGTM

**Pull request merged**